<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TraceWindowSettings</class>
 <widget class="QWidget" name="TraceWindowSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="uniqueTraces">
      <attribute name="title">
       <string>Traces</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RowColor">
      <attribute name="title">
       <string>Trace Color</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Trace row color</string>
         </property>
         <widget class="QLabel" name="traceLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>49</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Trace</string>
          </property>
         </widget>
         <widget class="QLabel" name="debugLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>49</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Debug</string>
          </property>
         </widget>
         <widget class="QLabel" name="infoLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>49</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Info</string>
          </property>
         </widget>
         <widget class="QPushButton" name="traceColorButton">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>30</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 255, 255);
selection-color: rgb(85, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="warningLabel">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>30</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Warning</string>
          </property>
         </widget>
         <widget class="QPushButton" name="warningColorButton">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>30</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 255, 255);
selection-color: rgb(85, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="errorLabel">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>60</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Error</string>
          </property>
         </widget>
         <widget class="QLabel" name="criticalLabel">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>90</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Critical</string>
          </property>
         </widget>
         <widget class="QPushButton" name="debugColorButton">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>60</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 255, 255);
selection-color: rgb(85, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="infoColorButton">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>90</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 255, 255);
selection-color: rgb(85, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="errorColorButton">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>60</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 255, 255);
selection-color: rgb(85, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="criticalColorButton">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>90</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 255, 255);
selection-color: rgb(85, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
